tests/tests_tui/test_tui_selectdirectorytree_modalscreen.py

```diff
@@ -0,0 +1,75 @@
+import platform
+from pathlib import Path
+
+import pytest
+
+from datashuttle.tui.app import TuiApp
+from datashuttle.tui.screens.modal_dialogs import (
+    SelectDirectoryTreeScreen,
+)
+
+
+@pytest.mark.asyncio
+async def test_select_directory_tree(monkeypatch):
+    """
+    Test that changing the drive in SelectDirectoryTreeScreen
+    updates the DirectoryTree path as expected.
+    """
+
+    # Set up fake and real drives
+    actual_drive = (
+        str(Path.home().drive) if platform.system() == "Windows" else "/"
+    )
+    fake_drive = "Z:\\" if platform.system() == "Windows" else "/mnt/fake"
+
+    monkeypatch.setattr(
+        SelectDirectoryTreeScreen,
+        "get_drives",
+        staticmethod(
+            lambda: [(actual_drive, actual_drive), (fake_drive, fake_drive)]
+        ),
+    )
+
+    app = TuiApp()
+    async with app.run_test(size=(200, 100)) as pilot:
+
+        await pilot.click("#mainwindow_new_project_button")
+
+        await pilot.click("#configs_local_path_select_button")
+        assert isinstance(pilot.app.screen, SelectDirectoryTreeScreen)
+
+        tree = pilot.app.screen.query_one(
+            "#select_directory_tree_directory_tree"
+        )
+        select = pilot.app.screen.query_one(
+            "#select_directory_tree_drive_select"
+        )
+
+        while not select.options:
```

Comment:
I was getting an error `select.options` was not found, I found removing this line worked